### ISSUE TITLE
homebrew game mr driller gameplay issues

### ISSUE THREAD

**Issue by dominater01** [hrydgard/ppsspp#15353]
im using version PPSSPP v1.12.3-781

the game runs like its on fast forward but i have tried to press fast forward and it just went faster, also the screen flashes maybe due to it going to fast

https://www.mediafire.com/file/hzh11l3olmeecku/DrillPSP.zip/file

**Comment by Gamemulatorer** [comment]
No issue on my phone.
![Uploading Screenshot_2022-01-27-09-03-01-873_org.ppsspp.ppsspp.jpg…]()

**Comment by dominater01** [comment]
https://www.youtube.com/watch?v=a_y2b-gcrQI

**Comment by Gamemulatorer** [comment]
What's your device?
Graphics backend?
Are you using default settings?

**Comment by dominater01** [comment]
PC
![pcspecs](https://user-images.githubusercontent.com/8470028/151287466-b523a8cf-a0a2-4a8a-9dd4-bd68297a8a36.png)
default settings since i tried a fresh version downloaded of the site just a few minutes ago
direct 3d 11

**Comment by ANR2ME** [comment]
may be the game supposed to run 30 FPS instead of 60 FPS? (ie. having similar timing issue to multiplayer on Fate Unlimited Codes, but in this case happened on single player)

**Comment by ANR2ME** [comment]
This game seems to call `sceDisplayWaitVblankStart` only once (at boot time)
![image](https://user-images.githubusercontent.com/7974720/151298111-3dae705b-c1ba-4b3a-a4fa-b5f004eb48fe.png)
Did this game tried to render as many frames as possible?

**Comment by Gamemulatorer** [comment]
Try to add this game to compat.ini

**[ForceMax60FPS]**
**# Mr Driller**
**PRIM02510 = true**

**Comment by ANR2ME** [comment]
i didn't have compat.ini file in my PSP\SYSTEM folder, so i created one and added those
But it have no difference LOL probably because the game wasn't supposed to run more than 30 FPS i think

**Comment by MojoJojoDojo** [comment]
For once it happens on single player lol

**Comment by iota97** [comment]
> i didn't have compat.ini file in my PSP\SYSTEM folder

It's supposed to be in the `assets/` folder next to the PPSSPP executable AFAIK, but I don't think it's relevant here anyway.

**Comment by benderscruffy** [comment]
using [v1.13.2-2144-g0682b7ba5](https://github.com/hrydgard/ppsspp/commit/0682b7ba5441ece466161b9bfb3dc9a549418d44)
this is still an issue

**Comment by Double-0-seven7** [comment]
https://github.com/hrydgard/ppsspp/issues/18242#issuecomment-1739917035

Another game got the same issue here but not homebrew or networking related.

**Comment by ANR2ME** [comment]
> [#18242 (comment)](https://github.com/hrydgard/ppsspp/issues/18242#issuecomment-1739917035)
> 
> Another game got the same issue here but not homebrew or networking related.

Yeah, Crazy Taxi's multiplayers (both the network one and the non-network one) running at 60 FPS instead of 30 FPS like in single player, may be both of them use the same code base which is different than single player code.
